druzhba/table.py

```diff
@@ -525,7 +525,7 @@ def row_generator(self):
         sql = self.get_query_sql()
         self._check_index_values()
         self.logger.info("Extracting %s table %s", self.db_name, self.source_table_name)
-        self.logger.debug("Running SQL: %s", sql)
+        self.logger.info("Running SQL: %s", sql)
```

Comment:
Was this change just for testing? I don't think we want this always-on

Reply:
I was thinking it would be useful to always have it at `info` in order to be able to see exactly what's being run.